src/components/Flows/__tests__/Flows.test.tsx

```
@@ -15,128 +15,1042 @@
+  //         </SessionProvider>
+  //       </GlobalContext.Provider>
+  //     );
+  //   });
+
+  //   // Should show timezone for cron schedules
+  //   expect(screen.getByText(/EST|EDT|PST|PDT|UTC/)).toBeInTheDocument();
+  // });
+
+  // ================================================================================
+  it('handles button disabled state with tempSyncState', async () => {
+    const mockUseSyncLock = require('@/customHooks/useSyncLock');
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)